jobs/JGDAS_ENKF_DIAG

```diff
@@ -91,28 +91,28 @@ export COMOUT="${ROTDIR}/enkf${CDUMP}.${PDY}/${cyc}"
 
 
 export ATMGES_ENSMEAN="${COMIN_GES_ENS}/${GPREFIX}atmf006${GSUFFIX}"
-if [ ! -f ${ATMGES_ENSMEAN} ]; then
+if [[ ! -f ${ATMGES_ENSMEAN} ]]; then
     echo "FATAL ERROR: FILE MISSING: ATMGES_ENSMEAN = ${ATMGES_ENSMEAN}"
     exit 1
 fi
 
 
 # Link observational data
 export PREPQC="${COMIN_OBS}/${OPREFIX}prepbufr"
-if [ ! -f ${PREPQC} ]; then
+if [[ ! -f ${PREPQC} ]]; then
     echo "WARNING: Global PREPBUFR FILE ${PREPQC} MISSING"
 fi
 export TCVITL="${COMIN_ANL}/${OPREFIX}syndata.tcvitals.tm00"
 if [[ ${DONST} = "YES" ]]; then
    if [[ ${MAKE_NSSTBUFR} == "YES" ]]; then
-      export NSSTBF="${COMOUT}/${OPREFIX}nsstbufr"
+      export NSSTBF="${COMIN_OBS}/${OPREFIX}nsstbufr"
    else
       export NSSTBF="${COMIN_OBS}/${OPREFIX}nsstbufr"
    fi
 fi
 
 if [[ ${MAKE_ACFTBUFR} == "YES" ]]; then
-   export PREPQCPF="${COMOUT}/${OPREFIX}prepbufr.acft_profiles"
+   export PREPQCPF="${COMIN_OBS}/${OPREFIX}prepbufr.acft_profiles"
```

Comment:
Same here. The conditional branches are identical, so can the conditional be removed?

Reply:
Try to keep same application logic to allow developer assign para path as needed. It can be removed at the end of v17 development cycle if there is no need for it.

Reply:
@lgannoaa 
Can you please remove the if-block here and only retain the `export PREPQCPF` line?  The if-else statements are the same, so there is no need for the if-block.

Reply:
Do you want me to remove the if statement in line 110?

Reply:
Yes.
If you look at the logic here, the if-else is executing the same thing.  If you keep the if, then the else case will not be executed.  That is an error.

Reply:
There is no else statement in this if logic on line 110.

Reply:
@lgannoaa 
The original code looked like this:
```
if [[ ${MAKE_ACFTBUFR} == "YES" ]]; then
   export PREPQCPF="${COMIN_OBS}/${OPREFIX}prepbufr.acft_profiles"
else
   export PREPQCPF="${COMIN_OBS}/${OPREFIX}prepbufr.acft_profiles"
fi
```
The `if-else` are the same.
If you only eliminate `else`, the result is:
```
if [[ ${MAKE_ACFTBUFR} == "YES" ]]; then
   export PREPQCPF="${COMIN_OBS}/${OPREFIX}prepbufr.acft_profiles"
fi
```
This means, when `MAKE_ACFTBUFR=NO`, there will be no `PREPQCPF` defined, which is what the `if-else` construct provided.